### PR TITLE
F034 updating home layout

### PR DESCRIPTION
**Pull request by jmaciasr**
Updating frontpage layout to delete old regions  and add "wide third" where is going be "nueva carrera tecnica". 

NOTE:
the implementation of  "panel-wide-courses" class, isn't so clear and I created one class to correct "panel-wide-home__courses".

### CODE REVIEW

File: sites/all/themes/cetaviano/layouts/cetav_frontpage/cetav-frontpage.tpl.php [hunk 1]
```diff
@@ -23,35 +23,20 @@
...
+    <div class="section-wide grid-width">
+      <?php print $content['wide_third']; ?>
     </div>
-  </div>
+  <?php endif ?>
   <?php if ($content['wide_fourth']): ?>
     <div class="section-wide slab-gray">
-      <div class="grid-width ">
+      <div class="grid-width">
       <?php print $content['wide_fourth']; ?>
```
Comment (mledezma) [inline sites/all/themes/cetaviano/layouts/cetav_frontpage/cetav-frontpage.tpl.php]: Identacion

File: sites/all/themes/cetaviano/sass/partials/components/_home-techs-intro.scss [hunk 2]
```diff
@@ -6,6 +6,11 @@
     width: 100%;
 }
 
+.panel-cetav-frontpage .panel-wide-home__courses {
```
Comment (mledezma) [inline sites/all/themes/cetaviano/sass/partials/components/_home-techs-intro.scss]: No hay mucha especificidad?

Comment (jmaciasr) [reply sites/all/themes/cetaviano/sass/partials/components/_home-techs-intro.scss]: Resuelve un problema bastante especifico, el padding de left y right para que no baje uno de los círculos en el home. 
La mando a algo más general?

**Comment by mledezma** [comment]
🚢

**Pull request merged**